EU4toV2/Source/V2World/Vic2TechSchool.h

```diff
@@ -52,13 +52,13 @@ class Vic2TechSchool: commonItems::parser
 		std::string getName() const { return name; }
 
 	private:
-		std::string name;
+		std::string name = "";
```

Comment:
std::string is an empty string by default, so need need to assign a value here.

Reply:
corrected.